Model/Menu.php

```diff
@@ -44,6 +76,7 @@ public function saveStores(array $stores)
             $connection->insert($table, ['menu_id' => $this->getId(), 'store_id' => $store]);
         }
         $connection->commit();
+        $this->cache->invalidatePageCache();
```

Comment:
I think we can also refactor this line so it's invalidated through a plugin?

Reply:
The plugin for this one is now implemented.